src/configs/redoc/livechat-customer-accounts-api/spec.yml

```diff
@@ -0,0 +1,249 @@
+openapi: 3.0.0
+info:
+  title: Customer Accounts API
```

Comment:
One general remark is that I think that the file name should also be "Customer Accounts API", without "LiveChat"

Reply:
I'll rename it